src/topics/engine/__fixtures__/fixtures.constants.ts

```diff
@@ -48,3 +48,8 @@ export const CE_NUMBER_TEXT = `ce : ${CE_NUMBER}`;
 
 export const PHYSICAL_STATE_IDENTIFIER = 'état physique';
 export const PHYSICAL_STATE_VALUE = 'liquide';
+
+export const STEAM_PRESSURE_TEMPERATURE = '50°c';
+export const STEAM_PRESSURE_IDENTIFIER_WITH_TEMPERATURE = `pression de vapeur (${STEAM_PRESSURE_TEMPERATURE})`;
+export const STEAM_PRESSURE_IDENTIFIER = `pression de vapeur`;
```

Comment:
On peut garder les backticks autour du string nécessitant un calcul à l'intérieur.
```suggestion
export const STEAM_PRESSURE_IDENTIFIER = 'pression de vapeur';
```

Reply:
yes petit oubli en dupliquant 🙈